src/api/Shrooms.Presentation.Api/Controllers/CommentController.cs

```diff
@@ -63,6 +63,11 @@ public async Task<IHttpActionResult> CreateComment(NewCommentViewModel comment)
                     notif.Notify(commentCreatedDto, userHubDto);
                 }, GetOrganizationName());
 
+                _asyncRunner.Run<NewMentionNotifier>(notif =>
+                {
+                    notif.NotifyNewMentionInComment(comment.PostId, commentDto.Mentions);
```

Comment:
Maybe it's worth checking if `commentDto.Mentions` is not empty before executing `asyncRunner`?